stubs/reportlab/@tests/stubtest_allowlist.txt

```diff
@@ -1,3 +1,28 @@
+# TODO: missing from stub
+error: reportlab.graphics.barcode.__all__ is not present in stub
```

Comment:
The allowlist entries need to be cleaned

Reply:
whoops